docs/src/modules/java/pages/replicated-entity.adoc

```diff
@@ -0,0 +1,192 @@
+= Implementing Replicated Entities
+
+include::ROOT:partial$include.adoc[]
+
+ifdef::review[REVIEWERS: this is the content from the previous version of replicated entities, please bring it up to date with the newest implementation]
+
+This page documents how to implement  Replicated Entities in Java. For information on what Replicated Entities are, please read how to https://developer.lightbend.com/docs/akka-serverless/designing/state-model.html[choose a state model] information first.
+
+A Replicated Entity can be created by annotating it with the link:{attachmentsdir}/api/com/akkaserverless/javasdk/replicatedentity/ReplicatedEntity.html[`@ReplicatedEntity`{tab-icon}, window="new"] annotation.
```

Comment:
I wonder if it's too early to add the documentation for replicated entity? I think the API is not finalized yet.

Reply:
We can either add docs for the current API, with annotations, and release that; or wait until this is all switched over to new codegen approach.